### PR TITLE
fix: Make the GPU warning waits for tutorial ends

### PR DESCRIPTION
**Pull request by sandrade-dcl**
Make the GPU warning waits for tutorial ends (if it is actived).

### CODE REVIEW

**Review by AjimenezDCL — APPROVED** [review]

File: kernel/packages/entryPoints/website.ts [hunk 1]
```diff
@@ -122,12 +122,12 @@ namespace webApp {
         i.ConfigureHUDElement(HUDElementID.PROFILE_HUD, { active: true, visible: true })
         i.ConfigureHUDElement(HUDElementID.USERS_AROUND_LIST_HUD, { active: voiceChatEnabled, visible: false })
         i.ConfigureHUDElement(HUDElementID.FRIENDS, { active: identity.hasConnectedWeb3, visible: false })
-        i.ConfigureHUDElement(HUDElementID.GRAPHIC_CARD_WARNING, { active: true, visible: true })
 
         EnsureProfile(identity.address)
           .then((profile) => {
             i.ConfigureEmailPrompt(profile.tutorialStep)
             i.ConfigureTutorial(profile.tutorialStep, HAS_INITIAL_POSITION_MARK)
+            i.ConfigureHUDElement(HUDElementID.GRAPHIC_CARD_WARNING, { active: true, visible: true })
```
Comment (pravusjif) [inline kernel/packages/entryPoints/website.ts:130]: this wouldn't make the warning to appear always by default?

Comment (AjimenezDCL) [reply kernel/packages/entryPoints/website.ts:130]: The GPU check is done in Unity (through the jslib) this just enables the whole process. For the next release we will disable it but right now we want to test that it's indeed, working

**Review by pravusjif — APPROVED** [review]

File: kernel/packages/entryPoints/website.ts (hunk 1 above)
Comment (pravusjif) [reply kernel/packages/entryPoints/website.ts:130]: awsome, thx

Comment (sandrade-dcl) [reply kernel/packages/entryPoints/website.ts:130]: @pravusjif, as @AjimenezDCL said, this is only for initialize the process but will not make the warning to appear always by default 👍

**Pull request merged**